xml/System.Net.Http/HttpCompletionOption.xml

```diff
@@ -36,7 +36,27 @@
   </Base>
   <Docs>
     <summary>Indicates if <see cref="T:System.Net.Http.HttpClient" /> operations should be considered completed either as soon as a response is available, or after reading the entire response message including the content.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
```

Comment:
The .NET docs guidelines mention an ongoing effort for all API documentation to converge into triple slash comments in source code: https://learn.microsoft.com/en-us/contribute/content/dotnet/api-documentation
However, I don't think the System.Net.Http namespace gone through this process yet, since there's no documentation in source code: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/System/Net/Http/HttpCompletionOption.cs.
Hence, I'm doing the edit directly on the backing XML documentation page.